src/factorial.js

```diff
@@ -0,0 +1,7 @@
+export default function factorial(num) {
+  var fact = 1;
+  for (var i = num; i > 1; i--) {
+  fact = fact * i;
```

Comment:
Nice job with this.
It does the exact same thing, but I've received feedback suggesting to replace line 4 with fact *= i